docs/examples/how-tos.md

```diff
@@ -0,0 +1,396 @@
+---
+category: examples
+order: 500
+modified_at: 2023-07-17
+---
+
+# How to's
+
+## Basics
+
+### How to set the height of CKEditor 5?
+
+The height of the editing area can be easily controlled with CSS.
+
+```css
+.ck.ck-content:not(.ck-comment__input *) {
```

Comment:
The `ck-content` class is a prefix for content styles (the same styles should be applied to the document rendered outside the editor. I'd suggest using `ck-editor__editable_inline` instead of `ck-content`. See the SO reply: https://stackoverflow.com/a/46559355

Reply:
Ok, to fix.